CHANGELOG.md

```diff
@@ -12,9 +12,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Added
+- Add `activeSpeakerPolicy` and `videoUplinkBandwidthPolicy` in meeting manager config to allow builders to pass in 
+  custom policies.
+- Allow passing in config in `meetingManager.join` to allow more flexibility.
```

Comment:
Should we re-word this to:
For more flexibility, allow passing `MeetingManagerConfig` to `meetingManager.join` method. Passing the config here would override config passed through `MeetingProvider` props.